java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -482,4 +482,24 @@ CompletableFuture<Map<String, String[]>> blmpop(
      */
     CompletableFuture<Map<String, String[]>> blmpop(
             String[] keys, PopDirection direction, double timeout);
+
+    /**
+     * Sets the list element at <code>index</code> to <code>element</code>. For details on the index
+     * argument, see {@link #lindex(String, long)}. Throws an exception for out of range <code>index
+     * </code>es.
```

Comment:
I think it is enough
```suggestion
     * Sets the list element at <code>index</code> to <code>element</code>.
```